tests/atest/transformers/RenameKeywords/expected/rename_pattern_partial.robot

```diff
@@ -47,6 +47,6 @@ All Upper Case
     Connect To System ABC
     Create Product DFG
 
-Underscores And.Dots
+Underscores And. Dots
```

Comment:
This fix also fixed the old issue where we removed the space from the keyword name after the dot, even if it's not possible to import library in the keyword name (so the space removal was not needed).